code/modules/mob/mob_defines.dm

```diff
@@ -86,9 +89,6 @@
 
 	var/research_scanner = 0 //For research scanner equipped mobs. Enable to show research data when examining.
 
-	var/list/grabbed_by = list(  )
-	var/list/requests = list(  )
```

Comment:
Removing two lists from /mob and only adding one var. A nice tradeoff :smiley_cat: